# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/linux24benh/82-raid5-niceer-output.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


I also sent this patch to the kernel hackers - because the output is
really confusing when the software RAID code does not choose the fastest
mode ... Rene

--- linuxi.orig/drivers/md/xor.c	Sun Sep 30 21:26:06 2001
+++ linux/drivers/md/xor.c	Fri Dec  7 18:16:10 2001
@@ -127,6 +127,8 @@
 
 #ifdef XOR_SELECT_TEMPLATE
 	fastest = XOR_SELECT_TEMPLATE(fastest);
+	printk("raid5: using %s to write around L2 cache\n",
+		fastest->name);
 #endif
 
 	active_template = fastest;
